components/director/internal/domain/formation/service.go

```diff
@@ -886,6 +934,16 @@ func (s *service) ResynchronizeFormationNotifications(ctx context.Context, forma
 		return errors.Wrapf(err, "while loading tenant from context")
 	}
 
+	// When the formation is in INITIAL state, we should not allow notifications to be sent
```

Comment:
The check was adapted and the comment was forgotten, it will be updated. Also the problem with delete formation is going to be addressed with the follow-up PR, as some e2e tests seem to delete formations which aren't empty. And the resynchronization of formation notifications with formation lifecycle notifications will be adapted in the follow up PR as well. This is just here temporarily.